docs_user/modules/proc_migrating-mon-from-controller-nodes-network.adoc

```diff
@@ -56,8 +56,8 @@ $ sudo os-net-config -c /etc/os-net-config/config.yaml
 $ ip -o a | grep 172.17.3.60
 ----
 
-. From the source node, ping the ip address that is migrated to the target node
-  and confirm that it is still reachable:
+. From the Ceph client node (controller-0), ping the ip address that is
+  migrated to the target node and confirm that it is still reachable:
```

Comment:
```suggestion
. From the Ceph client node, `controller-0`, ping the IP address that is
  migrated to the target node and confirm that it is still reachable:
```
Is "Ceph client node" the official name, or can it be "Red Hat Ceph Storage client node"?

Reply:
No, it's literally a node we use as a  client to interact with Ceph during the procedure, w/o requiring customers to jump from node to node.